code/modules/mob/living/carbon/human/dummy.dm

```diff
@@ -1,13 +1,16 @@
 
 /mob/living/carbon/human/dummy
 	real_name = "Test Dummy"
-	status_flags = GODMODE|CANPUSH
```

Comment:
did you mean to remove the `CANPUSH` flag ?

Reply:
Why does a dummy need can push

Reply:
i have no idea

Reply:
Keep in mind that it just inherits the status flags of the parent type in this case.